tsl/test/sql/remote_txn.sql

```diff
@@ -26,8 +26,8 @@ CREATE OR REPLACE FUNCTION add_loopback_server(
     database               NAME = current_database(),
     port                   INTEGER = inet_server_port(),
     if_not_exists          BOOLEAN = FALSE,
-    bootstrap		   BOOLEAN = TRUE,
-    bootstrap_database     NAME = 'postgres'
+    bootstrap			   BOOLEAN = TRUE,
+	password 			   TEXT = NULL
```

Comment:
Another odd indentation.

Reply:
This looks fine in my editor and must be a tab/whitespace thing. I'll try to clean it up